app/src/sharedTest/java/org/oppia/app/options/OptionsFragmentTest.kt

```diff
@@ -0,0 +1,297 @@
+import org.oppia.domain.profile.ProfileManagementController
+import org.oppia.domain.profile.ProfileTestHelper
+import org.oppia.util.logging.EnableConsoleLog
+import org.oppia.util.logging.EnableFileLog
+import org.oppia.util.logging.GlobalLogLevel
+import org.oppia.util.logging.LogLevel
+import org.oppia.util.threading.BackgroundDispatcher
+import org.oppia.util.threading.BlockingDispatcher
+import javax.inject.Inject
+import javax.inject.Qualifier
+import javax.inject.Singleton
+
+/** Tests for [OptionsFragment]. */
+@RunWith(AndroidJUnit4::class)
+class OptionsFragmentTest {
+
+
+  @Inject lateinit var profileTestHelper: ProfileTestHelper
+  @Inject lateinit var profileManagementController: ProfileManagementController
+  @Inject lateinit var context: Context
+
+  @Before
+  @ExperimentalCoroutinesApi
+  fun setUp() {
+    Intents.init()
+    setUpTestApplicationComponent()
+  }
+
+  @After
+  fun tearDown() {
+    Intents.release()
+  }
+
+  private fun setUpTestApplicationComponent() {
+    DaggerOptionsFragmentTest_TestApplicationComponent.builder()
+      .setApplication(ApplicationProvider.getApplicationContext())
+      .build()
+      .inject(this)
+  }
+
+  @get:Rule
+  var optionActivityTestRule: ActivityTestRule<OptionsActivity> = ActivityTestRule(
+    OptionsActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+  )
+
+  @Test
+  fun testOptionFragment_clickStoryTextSize_changeTextSizeToLargeSuccessfully() {
+    profileTestHelper.initializeProfiles()
+    ActivityScenario.launch(OptionsActivity::class.java).use {
+
+      onView(withId(androidx.preference.R.id.recycler_view))
+        .perform(
+          RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(
+            1,
+            click()
+          )
+        )
+      onView(withId(R.id.story_text_size_seekBar)).perform(clickSeekBar(10))
+    }
+  }
+
+  @Test
+  fun testOptionFragment_clickStoryTextSize_changeTextSizeToMediumSuccessfully() {
+    profileTestHelper.initializeProfiles()
+    ActivityScenario.launch(OptionsActivity::class.java).use {
+      onView(withId(androidx.preference.R.id.recycler_view))
+        .perform(
+          RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(
+            1,
+            click()
+          )
+        )
+      onView(withId(R.id.story_text_size_seekBar)).perform(clickSeekBar(5))
+    }
+  }
+
+  @Test
+  fun testOptionFragment_clickStoryTextSize_changeTextSizeToExtraLargeSuccessfully() {
+    profileTestHelper.initializeProfiles()
+    ActivityScenario.launch(OptionsActivity::class.java).use {
+      onView(withId(androidx.preference.R.id.recycler_view))
+        .perform(
+          RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(
+            1,
+            click()
+          )
+        )
+      onView(withId(R.id.story_text_size_seekBar)).perform(clickSeekBar(15))
+    }
+  }
+
+  private fun clickSeekBar(position: Int): ViewAction {
```

Comment:
This function should come after all the test cases

Reply:
Okay.